src/ResourceManager/HDInsight/Commands.HDInsight/JobCommands/GetAzureHDInsightJobOutputCommand.cs

```diff
@@ -133,25 +134,38 @@ private static string Convert(Stream stream)
 
         internal IStorageAccess GetDefaultStorageAccess(string resourceGroupName, string clusterName)
         {
+            var StorageAccountSuffix = "";
```

Comment:
the storageaccountsuffix is set for all of the built-in environments always.  It has nothing to do with the test context.  Using this value allows being compatible with existing offline scenarios for storage (i.e. when the user does not have access to storage management for the storage acocunt, but does have a key).